tests/test_integration.py

```diff
@@ -162,7 +162,7 @@ def test_supports_for_curses(self):
 		despair = active_skills[0]
 		assert despair.level == 22  # 20 + 2 (3 Empower)
 		assert despair.quality == 36  # 20 + 16 (3 Enhance)
-		assert despair.get_curse_effect() == 34  # 0 + 10 (20 20 Blasphemy Support) + 24 (20 20 Arrogance Support)
+		assert despair.get_curse_effect() == -7  # (100 + 24 (20 20 Arrogance Support)) * (100 - 25) (Blasphemy Support)
```

Comment:
wait this math does not check out. (100 + 24) * (100 - 25) isn't a negative number

Reply:
oh, 24% increased and 25% less is (1 + .24) * (1 - 0.25) = 0.93, which is equivalent to 7% reduced